sip-servlets-impl/src/main/java/org/mobicents/servlet/sip/core/dispatchers/SubsequentRequestDispatcher.java

```diff
@@ -190,11 +190,15 @@ public void dispatchMessage(final SipProvider sipProvider, SipServletMessageImpl
 							}				
 							return ;
 						} else {
+						    // Fixed github issue: https://github.com/RestComm/sip-servlets/issues/189
+						    // RFC 3261: https://tools.ietf.org/html/rfc3261#section-12.2.2
```

Comment:
i like this comment, please get accostume to write this kind of good stuff into your changes